collect_app/src/main/java/org/odk/collect/android/database/forms/DatabaseFormColumns.kt

```diff
@@ -24,7 +24,6 @@ object DatabaseFormColumns : BaseColumns {
     const val DISPLAY_SUBTEXT = "displaySubtext" // not used in the newest database version
     const val MD5_HASH = "md5Hash"
     const val DATE = "date"
-    const val MAX_DATE = "MAX(date)" // used only to get latest forms for each form_id
```

Comment:
This was confusing me (again) so I got rid of it. It's only used for one query so definitely shouldn't be referenced globally.

Reply:
Yeah we have a `HideOldVersionsTest` that checks this. I've not changed how the query works (it's still doing a `GROUP BY` and using `MAX`), I've just moved all of that into the provider and then used a projection map so that the rest of the app doesn't have to know about it.

Reply:
Great!